log/logger.go

```diff
@@ -18,20 +18,42 @@
 package log
 
 import (
+	"context"
+	"fmt"
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+	"golang.org/x/sys/unix"
 	"io"
 	"os"
+	"strings"
 	"sync"
 	"time"
 
 	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
+	"gopkg.in/natefinch/lumberjack.v2"
 )
 
+const ObkvTraceIdName = "_ObkvTraceIdName"
+
 var (
 	globalMutex         sync.Mutex
-	defaultGlobalLogger = NewLogger(os.Stderr, InfoLevel, AddCaller())
+	defaultGlobalLogger *Logger
+	TraceId             string
+	LogType             = "Default"
```

Comment:
why not const variable ？

Reply:
logtype will change in log print, 4096 BufferSize has change